content/en/docs/setup/install/multicluster/shared/index.md

```diff
@@ -14,11 +14,13 @@ test: no
 ---
 
 Setup a [multicluster Istio service mesh](/docs/ops/deployment/deployment-models/#multiple-clusters)
-across multiple clusters with a shared control plane. In this configuration, multiple Kubernetes clusters running
-a remote configuration connect to a shared Istio [control plane](/docs/ops/deployment/deployment-models/#control-plane-models)
-running in a main cluster. Clusters may be on the same network or different networks than other
-clusters in the mesh. Once one or more remote Kubernetes clusters are connected to the Istio control plane,
-Envoy can then form a mesh.
+across multiple {{< gloss "remote cluster" >}}remote clusters{{< /gloss >}} that
+share a control plane. In this configuration, multiple Kubernetes clusters
+running a remote configuration connect to a shared Istio [control plane](/docs/ops/deployment/deployment-models/#control-plane-models)
+running in a {{< gloss >}}primary cluster{{< /gloss >}}. The remote clusters can
+share the same network or be in other networks. After one or more remote
```

Comment:
After one or more remove clusters are connected, the control plane will manage the service mesh across all service endpoints in the system. (i.e. get rid of the mention of Envoy)